docs/api_reference.rst

```diff
@@ -0,0 +1,7 @@
+fileinfo module
+===============
+
```

Comment:
```suggestion

.. automodule:: surfactant
   :members:
   :undoc-members:
   :show-inheritance:
   
```

Reply:
I'm thinking it makes sense to combine these onto a single API Reference page, given that the public part of the API may be relatively small.